STEP_Project/tests/popupTest.js

```diff
@@ -36,26 +36,33 @@ describe('Testing pageType that determines enabling/disabling of buttons dependi
 /**
  * testing function disableButtons
  */
+function initTest()
+{
+  const arButton = document.createElement('BUTTON')
+  arButton.setAttribute("id", "addRemove");
+  const sButton = document.createElement('BUTTON')
+  sButton.setAttribute("id", "select");
+  document.body.appendChild(arButton);
+  document.body.appendChild(sButton);
```

Comment:
Suggestion was to move only this part in a separate function, let's say createDom (as you did for removing DOM elements). We can keep the initialization of select and remove buttons in the beforeEach method itself.

Reply:
updated function modularization